metashare/api/jobs.py

```diff
@@ -161,3 +172,11 @@ def create_branches_on_github_then_create_scratch_org(
 create_branches_on_github_then_create_scratch_org_job = job(
     create_branches_on_github_then_create_scratch_org
 )
+
+
+def delete_scratch_org(scratch_org):
+    sf_run_flow.delete_scratch_org(scratch_org)
```

Comment:
This should run inside an error handler (similar to `report_errors_on`). On a deletion error, we should:

- set `deletion_queued_at` back to `None` on the model, and save it (*not* deleting the model)
- then emit a WS event: `{ type: 'SCRATCH_ORG_DELETE_FAILED', payload: { message: <string>, model: <Org> } }` (this should happen after, so that `deletion_queued_at` is `null` in the serialized WS payload)